frontend/src/utils/emails/create-email-link.ts

```diff
@@ -0,0 +1,35 @@
+import { assert } from 'console'
```

Comment:
Can't resolve module 'console'. Might be a part of Node

Reply:
@coreball I basically tested it by running the typescript file as a script. That's what the changes in the ts-config are for.

Reply:
Oh ya that's why, runtime environments are different